mne/beamformer/tests/test_lcmv.py

```diff
@@ -698,72 +698,6 @@ def test_lcmv_ctf_comp():
         make_lcmv(info_comp, fwd, data_cov)
 
 
-@testing.requires_testing_data  # formula does not work for vector!
-@pytest.mark.parametrize('pick_ori', ['max-power', 'normal'])
-@pytest.mark.parametrize('reg', (0.05, 0.))
-def test_unit_noise_gain_formula(pick_ori, reg):
-    """Test unit-noise-gain filter against formula."""
-    # The unit-noise-gain beamformer is computed following a shortcut, where
-    # W_ung = inv(sqrt(W_ug @ W_ug.T)) @ W_ug
-    # with W_ung referring to the unit-noise-gain (weight normalized) filter
-    # and W_ug referring to the above-calculated unit-gain filter stored in W.
-    # Here, we check that the formulation in Sekihara & Nagarajan still holds.
-    raw = mne.io.read_raw_fif(fname_raw, preload=True)
-    events = mne.find_events(raw)
-    raw.pick_types(meg='mag')
-    # assert len(raw.ch_names) == 305
-    epochs = mne.Epochs(raw, events, None, preload=True)
-
-    # with pytest.warns(RuntimeWarning, match='Too few samples'):
-    data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15)
-
-    noise_cov = None  # no whitening to make things easier
-    forward = mne.read_forward_solution(fname_fwd)
-    convert_forward_solution(forward, surf_ori=True, copy=False)
-
-    # compute the unit-noise-gain scalar beamformer
-    rank = None
-    # TODO: Decide if we should actually use unit-noise-gain here...
-    filters = make_lcmv(epochs.info, forward, data_cov, reg=reg,
-                        noise_cov=noise_cov, pick_ori=pick_ori,
-                        weight_norm='unit-noise-gain', rank=rank)
-
-    # manipulate forward to have same orientation picking as above
-    forward = mne.pick_types_forward(forward, meg='mag')  # restrict to MAG
-    fwd_sol = forward['sol']['data'].T
-    n_sources, n_channels = forward['nsource'], forward['nchan']
-    fwd_sol = np.reshape(fwd_sol, (n_sources, 3, n_channels))
-    fwd_sol = fwd_sol.transpose(0, 2, 1)
-    if pick_ori == 'normal':
-        fwd_sol = fwd_sol[:, :, 2:3]
-        n_orient = 1
-    elif pick_ori == 'max-power':
-        fwd_sol = np.matmul(fwd_sol, filters['max_power_ori'][..., np.newaxis])
```

Comment:
@larsoner I am referring to this for a test of the `max_power_ori` application in `_compute_beamformer`